src/main/java/dk/cngroup/lentils/controller/TeamController.java

```diff
@@ -27,20 +24,20 @@ public TeamController(TeamService teamService) {
         this.teamService = teamService;
     }
 
-    @GetMapping(value = "/add")
+    @GetMapping(value = "")
```

Comment:
Můžeš ještě mrknout, jestli nejde napsat jen @GetMapping místo @GetMapping(value = "")?

Reply:
jde to - upraveno